lib/index.js

```diff
@@ -11,5 +12,6 @@ var main = function (options, callback) {
 
 module.exports = main;
 module.exports.launch = main;
+module.exports.argv = argv;
```

Comment:
The point is that saucie's `launch` function doesn't parse args so [this](https://github.com/johanneswuerbach/ember-cli-sauce/blob/master/lib/commands/sauce-launch.js#L14) doesn't end up actually passing the args (unless I'm missing something). Also I'm not sure it's a good idea to pass command line args when a function of saucie's API is invoked (because like I said above you might be seeing command line option passing errors when invoking a function of another node module which would be pretty unexpected).